addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -336,10 +336,6 @@ function classToStyle($editable, cssRules) {
                 writes.push(() => { node.style[styleName] = ''; });
             }
         }
-        // Ignore font-family (mail-safe font declared in <head>)
-        if ('font-family' in css) {
-            delete css['font-family'];
-        }
```

Comment:
So we *do* accept not mail-safe fonts now, right ?

Reply:
No but we can now change to font to another mail-safe font.